### PR TITLE
Release v0.11.3

### PR DESCRIPTION
**Pull request by AurevoirXavier**

### CODE REVIEW

**Comment by AurevoirXavier** [comment]
## Darwinia

[fork.json.tar.zstd.zip](#)

### `fork-off-substrate`
<details>
<summary>Click to expand!</summary>

```
```
</details>

### Try Runtime
<details>
<summary>Click to expand!</summary>

```
2021-08-22 12:37:58 `pallet_timestamp::UnixTime::now` is called at genesis, invalid value returned: 0
2021-08-22 12:37:58 [#0] 🗳  Finalized election round with compute ElectionCompute::OnChain.
2021-08-22 12:37:58 [0] 💸 new validator set of size 1 has been processed for era 1
2021-08-22 12:37:58 initializing remote client to "http://localhost:20000"
2021-08-22 12:37:58 scraping keypairs from remote node http://localhost:20000 @ 0xd565d098c802752a3a2d2de2270f9d8e1713fb541b1dbaffdaa116cd677b19f8
2021-08-22 12:37:58 downloading data for all modules.
2021-08-22 12:39:44 extending externalities with 1 manually injected keys
2021-08-22 12:39:44 injecting a total of 103586 keys
2021-08-22 12:39:46 try-runtime::on_runtime_upgrade Darwinia.
2021-08-22 12:39:46 `ProposalCount` Removed
2021-08-22 12:39:46 `Approvals` Removed
2021-08-22 12:39:46 `Proposals` Migrated
2021-08-22 12:39:46 `Approvals` Migrated
2021-08-22 12:39:46 `Tips` Migrated
2021-08-22 12:39:46 `BountyCount` Migrated
2021-08-22 12:39:46 `Bounties` Migrated
2021-08-22 12:39:46 `BountyDescriptions` Migrated
2021-08-22 12:39:46 `BountyApprovals` Migrated
2021-08-22 12:39:46 `ProposalCount` Removed
2021-08-22 12:39:46 `Approvals` Removed
2021-08-22 12:39:46 `Proposals` Migrated
2021-08-22 12:39:46 `Approvals` Migrated
2021-08-22 12:39:46 `Tips` Migrated
2021-08-22 12:39:46 `BountyCount` Migrated
2021-08-22 12:39:46 `Bounties` Migrated
2021-08-22 12:39:46 `BountyDescriptions` Migrated
2021-08-22 12:39:46 `BountyApprovals` Migrated
2021-08-22 12:39:46 ⚠️ System declares internal migrations (which *might* execute), setting storage version to PalletVersion { major: 3, minor: 0, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for TronBacking, setting storage version to PalletVersion { major: 2, minor: 6, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for EthereumRelayAuthorities, setting storage version to PalletVersion { major: 2, minor: 6, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for EthereumRelayerGame, setting storage version to PalletVersion { major: 2, minor: 6, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for EthereumBacking, setting storage version to PalletVersion { major: 2, minor: 6, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for EthereumRelay, setting storage version to PalletVersion { major: 2, minor: 6, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for CrabBacking, setting storage version to PalletVersion { major: 2, minor: 5, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for Multisig, setting storage version to PalletVersion { major: 3, minor: 0, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for Proxy, setting storage version to PalletVersion { major: 3, minor: 0, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for Scheduler, setting storage version to PalletVersion { major: 3, minor: 0, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for Recovery, setting storage version to PalletVersion { major: 3, minor: 0, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for Society, setting storage version to PalletVersion { major: 3, minor: 0, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for Identity, setting storage version to PalletVersion { major: 3, minor: 0, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for Utility, setting storage version to PalletVersion { major: 3, minor: 0, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for Vesting, setting storage version to PalletVersion { major: 2, minor: 6, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for Sudo, setting storage version to PalletVersion { major: 3, minor: 0, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for Bounties, setting storage version to PalletVersion { major: 3, minor: 0, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for Tips, setting storage version to PalletVersion { major: 3, minor: 0, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for KtonTreasury, setting storage version to PalletVersion { major: 3, minor: 0, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for Treasury, setting storage version to PalletVersion { major: 3, minor: 0, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for TechnicalMembership, setting storage version to PalletVersion { major: 3, minor: 0, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for PhragmenElection, setting storage version to PalletVersion { major: 2, minor: 6, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for TechnicalCommittee, setting storage version to PalletVersion { major: 3, minor: 0, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for Council, setting storage version to PalletVersion { major: 3, minor: 0, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for Democracy, setting storage version to PalletVersion { major: 2, minor: 6, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for DarwiniaHeaderMMR, setting storage version to PalletVersion { major: 2, minor: 6, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for AuthorityDiscovery, setting storage version to PalletVersion { major: 3, minor: 0, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for ImOnline, setting storage version to PalletVersion { major: 3, minor: 0, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for Grandpa, setting storage version to PalletVersion { major: 3, minor: 0, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for Session, setting storage version to PalletVersion { major: 3, minor: 0, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for Historical, setting storage version to PalletVersion { major: 3, minor: 0, patch: 0 }
2021-08-22 12:39:46 ⚠️ Offences declares internal migrations (which *might* execute), setting storage version to PalletVersion { major: 3, minor: 0, patch: 0 }
2021-08-22 12:39:46 have 0 deferred offences, applying.
2021-08-22 12:39:46 ✅ no migration for Staking, setting storage version to PalletVersion { major: 2, minor: 6, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for ElectionProviderMultiPhase, setting storage version to PalletVersion { major: 3, minor: 0, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for Authorship, setting storage version to PalletVersion { major: 3, minor: 0, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for TransactionPayment, setting storage version to PalletVersion { major: 3, minor: 0, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for Kton, setting storage version to PalletVersion { major: 2, minor: 6, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for Balances, setting storage version to PalletVersion { major: 2, minor: 6, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for Timestamp, setting storage version to PalletVersion { major: 3, minor: 0, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for Babe, setting storage version to PalletVersion { major: 3, minor: 0, patch: 0 }
2021-08-22 12:39:46 ✅ no migration for RandomnessCollectiveFlip, setting storage version to PalletVersion { major: 3, minor: 0, patch: 0 }
2021-08-22 12:39:46 try-runtime executed without errors. Consumed weight = 513081600000000, total weight = 512000000000000 (1.0021125)
```
</details>

**Review by HackFisher — APPROVED** [review]
Let's merge and tag first, still waiting the fork-off-substrate test result.

**Comment by HackFisher** [comment]
missing change log

**Review by HackFisher — CHANGES_REQUESTED** [review]
https://github.com/darwinia-network/darwinia/pull/705 Move DarwiniaCrabIssuing to CrabIssuing

This migration should not be deleted?

**Comment by AurevoirXavier** [comment]
> Let's merge and tag first, still waiting the fork-off-substrate test result.

Exporting.

**Review by HackFisher — APPROVED** [review]

**Comment by HackFisher** [comment]
Crab's try_runtime might need to be re-run since the DarwiniaCrabIssuing to CrabIssuing was add back.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,65 @@
...
+| General Linux | x86_64 |       2.23       |       4.0       |     ✔︎     |
+
+#### Docker
+```docker
+docker pull quay.io/darwinia-network/darwinia:v0.11.3
+```
+
+#### Update(s)
+- https://github.com/darwinia-network/darwinia-common/pull/747 Pre-Verify for Ethereum Relay
+- https://github.com/darwinia-network/darwinia-common/pull/760 A dispatch call for internal transact
```
Comment (AsceticBear) [inline CHANGELOG.md]: A missing update: Drop `darwinia-cli` (#706)

**Comment by HackFisher** [comment]
BTW, what't the changes in substrate?

Did we check the migration in substrate, and what's the companion release in Polkadot for branch darwinia-v0.11.3

https://github.com/darwinia-network/substrate/compare/darwinia-v0.11.2...darwinia-v0.11.3

File: CHANGELOG.md (hunk 1 above)
Comment (AurevoirXavier) [reply CHANGELOG.md]: Only post the update that is worth noting.

**Comment by AurevoirXavier** [comment]
> BTW, what's the changes in substrate?
> 
> Did we check the migration in substrate, and what's the companion release in Polkadot for branch darwinia-v0.11.3
> 
> [darwinia-network/substrate@darwinia-v0.11.2...darwinia-v0.11.3](https://github.com/darwinia-network/substrate/compare/darwinia-v0.11.2...darwinia-v0.11.3)

There's no substrate. I will update the tag. This release was an unexpected one. v0.11.3 is another update.

**Comment by HackFisher** [comment]
Remember to update the substrate dep in the next tag PR.

**Pull request merged**